packages/swc-helpers/src/_extends.mjs

```diff
@@ -1,21 +1,14 @@
-function extends_() {
-  extends_ = Object.assign || function (target) {
-    for (var i = 1; i < arguments.length; i++) {
-      var source = arguments[i];
-
-      for (var key in source) {
-        if (Object.prototype.hasOwnProperty.call(source, key)) {
-          target[key] = source[key];
+export function _extends() {
+    _extends = Object.assign || function(target) {
```

Comment:
I cannot rewrite the exported function.
Due to an incorrect translation during the transpilation to CJS, what is your perspective? @kdy1 

https://github.com/swc-project/swc/blob/3f2ba68d4b4b560a2a1e13ba92919fe54a0d3413/crates/swc_ecma_transforms_module/src/common_js.rs#L271-L282

Reply:
I think we can remove this 'optimization' if required

Reply:
Do you lean towards breaking changes?
Do you prefer named imports?

If it is necessary to introduce a breaking changes, there exists an opportunity for refinement.
This means 0.5.x for `@swc/helpers`

Reply:
I don't _want_ breaking changes, but if it's inevitable I'm fine with it.

Reply:
The breaking change refers to renaming the function name of exporting the index to `snake_case`.
However, this entry has never been used in our transpilation except for importing the helper in scripts environment (issue #7144). However, it is already broken.

Reply:
Then I think it would be okay to have a breaking change of `@swc/helpers`.